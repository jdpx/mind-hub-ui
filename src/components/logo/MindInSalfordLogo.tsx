import React from 'react'

import logo from '../../assets/mind-in-salford-logo.png'

export default function MindInSalfordLogo() {
    return <img src={logo} alt="Mind In Salford Logo" height={60} />
}
