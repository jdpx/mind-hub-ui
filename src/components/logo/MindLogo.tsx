import React from 'react'

import logo from '../../assets/mind-logo.svg'

export default function MindLogo() {
    return <img src={logo} alt="Mind Logo" height={60} />
}
