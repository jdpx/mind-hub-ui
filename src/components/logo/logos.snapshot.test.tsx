import React from 'react'
import { render } from '@testing-library/react'

import MindInSalfordLogo from './MindInSalfordLogo'
import MindLogo from './MindLogo'

describe('logos Snapshots', () => {
    it('renders Mind In Salford logo', () => {
        const { asFragment } = render(<MindInSalfordLogo />)
        expect(asFragment()).toMatchSnapshot()
    })

    it('renders Mind logo', () => {
        const { asFragment } = render(<MindLogo />)
        expect(asFragment()).toMatchSnapshot()
    })
})
