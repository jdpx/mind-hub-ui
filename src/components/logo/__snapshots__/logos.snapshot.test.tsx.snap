// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`logos Snapshots renders Mind In Salford logo 1`] = `
<DocumentFragment>
  <img
    alt="Mind In Salford Logo"
    height="60"
    src="mind-in-salford-logo.png"
  />
</DocumentFragment>
`;

exports[`logos Snapshots renders Mind logo 1`] = `
<DocumentFragment>
  <img
    alt="Mind Logo"
    height="60"
    src="mind-logo.svg"
  />
</DocumentFragment>
`;
